### DISCUSSION TITLE
[qrcode] is missing maskPattern in  QRCodeRenderersOptions

### DISCUSSION THREAD

**Discussion by jo-soft** [DefinitelyTyped/DefinitelyTyped#61735] [Issues with a @types package]
There is one option missing for creating QR codes:

https://github.com/soldair/node-qrcode/blob/master/lib/core/qrcode.js#L487 shows uses the option `maskPattern` which gets passed in the options object all the way e.g.  from the `toCanvas` function.

from the  documentations: 

> Type: Number
> 
> Mask pattern used to mask the symbol.
> Possible values are 0, 1, 2, 3, 4, 5, 6, 7.
> If not specified the more suitable value will be calculated.

[*] https://github.com/soldair/node-qrcode

[a comment by an automated account is not shown]

**Comment by plantain-00** [comment]
You can create a PR to improve it.

**Reply by jo-soft** [reply]
done: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61746
